### PR TITLE
fiona/ add deriv x FAQs to help center

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

- deriv x FAQs section added

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — COMMENTED** [review]
Hi @fiona-deriv check on this inactive hyperlink _'What markets can I trade on Deriv X?'_. Thanks

**Comment by fiona-deriv** [comment]
@arome-binary this is still a WIP that's why haven't updated the card yet

File: src/pages/help-centre/_help-articles.js [hunk 1]
```diff
@@ -238,6 +238,89 @@ export const articles = [
...
+                title: <Localize translate_text="What markets can I trade on Deriv X?" />,
+                category: 'Deriv x',
+                sub_category: <Localize translate_text="Markets" />,
+                label: 'markets-on-deriv-x',
+            },
+            {
+                title: <Localize translate_text="What is the minimum and maximum amount to trade on Deriv X?" />,
+                category: 'Deriv x',
+                sub_category: <Localize translate_text="How to trade" />,
+                label: 'minimum-and-maximum-amoount-to-trade',
```
Comment (ShuvoHabib) [inline src/pages/help-centre/_help-articles.js:266]: Spelling mistake on label

File: src/pages/help-centre/_help-articles.js [hunk 2]
```diff
@@ -238,6 +238,89 @@ export const articles = [
...
+            },
+            {
+                title: <Localize translate_text="What is the major difference between DTrader, Deriv MT5(DMT5) and Deriv X?" />,
+                category: 'Deriv x',
+                sub_category: <Localize translate_text="Platforms" />,
+                label: 'differences-of-dtrader-dmt5-deriv-x',
+            },
+            {
+                title: <Localize translate_text="How do I create a deriv x account?" />,
```
Comment (ShuvoHabib) [inline src/pages/help-centre/_help-articles.js:275]: Should this Deriv X be in capital case? Please confirm with the design

Comment (ShuvoHabib) [inline src/pages/help-centre/_help-articles.js:275]: Should this Deriv X be in capital case? Please confirm with the design

File: src/pages/help-centre/_help-articles.js [hunk 3]
```diff
@@ -238,6 +238,89 @@ export const articles = [
...
+                label: 'differences-of-trading-and-deriv-password',
+            },
+            {
+                title: <Localize translate_text="How do I reset my Deriv X password?" />,
+                category: 'Deriv x',
+                sub_category: <Localize translate_text="Platforms" />,
+                label: 'reset-deriv-x-password',
+            },
+            {
+                title: <Localize translate_text="Where can I find my deriv X account information?" />,
```
Comment (ShuvoHabib) [inline src/pages/help-centre/_help-articles.js:305]: Should this Deriv X be in capital case? Please confirm with the design

File: src/pages/help-centre/deriv-x.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+                    label="minimum-and-maximum-amount-to-trade"
+                />
+                <DifferenceDMT5DTraderDerivX
+                    text={localize(
+                        'What is the major difference between DTrader, Deriv MT5(DMT5) and Deriv X?',
+                    )}
+                    label="differences-of-dtrader-dmt5-deriv-x"
+                />
+                <DerivXAccount
+                    text={localize('How do I create a deriv x account?')}
```
Comment (ShuvoHabib) [inline src/pages/help-centre/deriv-x.js:146]: Should this Deriv X be in capital case? Please confirm with the design

File: src/pages/help-centre/_help-articles.js (hunk 1 above)
Comment (fiona-deriv) [reply src/pages/help-centre/_help-articles.js:266]: I'll resolve this in the next commit

**Pull request closed without merging**